econml/tests/test_treatment_featurization.py

```diff
@@ -538,6 +556,22 @@ def weird_func(x):
         # depending on sklearn version, bad feature names either throws error or only uses first relevant name
         assert est.cate_treatment_names(['too', 'many', 'feature_names']) in [None, ['too', 'too^2']]
 
+    def test_alpha_passthrough(self):
+        X = np.random.normal(size=(100, 3))
+        T = np.random.normal(size=(100, 1)) + X[:, [0]]
+        Y = np.random.normal(size=(100, 1)) + T + X[:, [0]]
+
+        est = LinearDML(model_y=LinearRegression(), model_t=LinearRegression(),
+                        treatment_featurizer=FunctionTransformer())
+        est.fit(Y=Y, T=T, X=X)
+
+        # ensure alpha is passed
```

Comment:
This seems like an okay test; you might alternatively consider specifying two different values of alpha and confirming that the corresponding two intervals are nested correctly.

Reply:
Done